### PR TITLE
Added a help command

### LINKED ISSUE #32: Help command
**Is your feature request related to a problem? Please describe.**
There isn't a way for a regular user to see a list of possible Horace commands.

**Describe the solution you'd like**
A simple `!help` command will probably suffice, as long as it's regularly updated. Maybe some optional args like `dm` in the Pomodoro bot.

### PR DESCRIPTION
**Pull request by Zaph-x**
This pull request will add a help command to the bot. This will implement the feature resqeted in-, and close #32.

The command allows the user to list every command or get the specific usage of a specific command. The help messages are sent as private messages, to minimise clutter in the chat
helplist
help for command

Furthermore, if a command is misspelled, when asking for a specific command, the bot will suggest the most plausible match of the misspelled command, using Levenshtein Distance.
suggestion of misspelled command

### CODE REVIEW

**Comment by Zaph-x** [comment]
Hmm seems it won't pass the ci as it expects tab instead of spaces. I'll correct this in a commit coming soon

**Comment by Dendrobyte** [comment]
You should be able to run the linter, might help ya out! Thanks for the
help Zaph :D

**Comment by Moesgaarda** [comment]
> 
> 
> Hmm seems it won't pass the ci as it expects tab instead of spaces. I'll correct this in a commit coming soon

Thanks for your contributions so far! There's an npm script called fixlinting which should fix it without you having to do much. 
Your solution looks elegant, nice idea to include Levenshtein for unknown commands. 👍

**Comment by Zaph-x** [comment]
All tests now pass in the CI. Should be done and ready to implement

File: commands/help.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+			console.error(err);
+		});
+	} else if (args.length === 1) {
+		let command = commands.find(command => command.config.name === args[0].toLowerCase()
+            || command.config.aliases.find(alias => alias === args[0].toLowerCase()));
+
+		if (command) {
+			let helpMessage = [`__**Help for the ${command.config.name} command:**__\n\n`];
+			helpMessage.push(command.config.description, '\n\n');
+			helpMessage.push('Aliasses: ', command.config.aliases.map(alias => '`' + alias + '`').join(', '), '\n');
```
Comment (Moesgaarda) [inline commands/help.js]:
```suggestion
			helpMessage.push('Aliases: ', command.config.aliases.map(alias => '`' + alias + '`').join(', '), '\n');
```

File: commands/facebook.js [hunk 2]
```diff
@@ -7,4 +7,6 @@ module.exports.execute = async (client, message) => {
 module.exports.config = {
 	name: 'facebook',
 	aliases: ['facebook', 'fb'],
+	description: 'I will send you the link to our facebook group.',
```
Comment (Moesgaarda) [inline commands/facebook.js]:
```suggestion
	description: 'I will send you the link to our Facebook group.',
```

**Comment by Zaph-x** [comment]
That should be corrected now :)

**Pull request merged**